src/blocks/ActionBlock.tsx

```diff
@@ -15,7 +16,7 @@ interface ActionBlockProps {
   actions: React.ReactNode[]
   background?: string
   className?: string
-  header: string
+  header: HeadingProps
```

Comment:
I'm not a huge fan of this style since the syntax means the passed in values are an object rather than a component.  e.g `header={{ children: "Action Block" }}`. In my opinion it's weird to pass `children` as a sub-prop. 

Would it be possible to change this to `header: React.ReactElement<HeadingProps>` and then when you call ActionBlock it would look like `<ActionBlock header={<Heading .... />} .../>` the hard part of this way though is to add the `action-block__header` class to it.

I know this isn't a long-term solution since I don't know if ActionBlock is getting ported over to ui-seeds. So I'm ok with this change as is, but since this is a breaking change I wonder if we want to think about the best practice for this sort of component

Reply:
It's tricky because I think ideally I'd like to be strict with consumers about what could be passed into a heading prop like that, because React.ReactElement<HeadingProps> looks to take in any JSX, but I definitely agree with passing in a component, I've updated it to React.ReactNode!